monitor/check_workflows.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+from datetime import datetime
+import argparse
+import json
+import string
+import requests
+
+workflow_names = ['AdapterSmartSeq2SingleCell', 'SmartSeq2SingleCell', 'RunHisat2RsemPipeline', 'RunHisat2Pipeline', 'total']
+ignore_before = '2018-03-01T00:00:00.000Z'
+
+def run(cromwell_url, user, password):
+    auth = requests.auth.HTTPBasicAuth(user, password)
+
+    now = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
nit: 
```
from datetime import datetime
from pytz import reference

datetime.now().strftime("%a, %Y-%m-%d %H:%M:%S, " + reference.LocalTimezone().tzname(datetime.now()))
```
which will provide info like this: `Fri, 2018-03-02 17:26:35, EST`

Reply:
Hmm, I see your point, however we have other logs that are in GMT, such as in Cromwell so we probably want to be consistent here. We can change in future if it becomes problematic.